bootstrap/components/sinks/kafka-sink-topology-component.json

```diff
@@ -35,13 +35,22 @@
         "hint": "schema,override"
       },
       {
-        "uiName": "Writer schema version",
+        "uiName": "Writer Schema Branch",
+        "fieldName": "schemaBranch",
+        "isOptional": false,
+        "tooltip": "Schema branch to be used fetching for writer schema versions. Default branch is MASTER",
+        "type": "enumstring",
+        "options": [],
+        "hint": "dependOn-topic,schemaBranch"
```

Comment:
@satishd Need a minor change, the hint for " writer schema branch" is missing an 's' in "dependsOn-topic", hence I got an issue on UI. Could you make this change and verify ?

Reply:
@joylyn Pushed the required change in latest commit.